### PR TITLE
First pass at Features Matrix

### PR DESCRIPTION
**Pull request by david-martin**
I would like to get thoughts on us maintaining a features matrix for our sdks.

@josemigallas I put this together based on a disucssion on IRC about where we are headed with the SDKs and if we have a Spec.
I think this might help with answering part of your questions.

I've pinged some others to get their take on this

### CODE REVIEW

File: sdks/sdk_features_matrix.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+== Features Matrix
+
+| Feature	| Android	| iOS	| Cordova	|
+|:--	|:-:	|:-:	|:-:	|:-:	|
+| config	| X	| X	| X		|
+| login	| X	| X	|   	|
+| logout	| X	| X	|   	|
+| checkStatus	| X	| X	|   	|
+| self defense checks	| X	|   	|   	|
```
Comment (wei-lee) [inline sdks/sdk_features_matrix.md]: I think these are done for ios

File: sdks/sdk_features_matrix.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+* Cordova (JS) - https://github.com/aerogear/aerogear-js-sdk
+
+== Features Matrix
+
+| Feature	| Android	| iOS	| Cordova	|
+|:--	|:-:	|:-:	|:-:	|:-:	|
+| config	| X	| X	| X		|
+| login	| X	| X	|   	|
+| logout	| X	| X	|   	|
+| checkStatus	| X	| X	|   	|
```
Comment (wei-lee) [inline sdks/sdk_features_matrix.md:17]: I think login/logout/checkStatus are done for iOS

File: sdks/sdk_features_matrix.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+| self defense checks	| X	|   	|   	|
+| self defense checks metrics	| X	|   	|   	|
+| SSO	| X	|   	|   	|
```
Comment (wei-lee) [inline sdks/sdk_features_matrix.md:20]: this is done for iOS as well

File: sdks/sdk_features_matrix.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+= AeroGear SDK Feature Matrix
```
Comment (wei-lee) [inline sdks/sdk_features_matrix.md:1]: Is the proposal the right place for it? Feels like this should be in the document repo.

File: sdks/sdk_features_matrix.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+
+== SDKs
+
+* Android (Native) - https://github.com/aerogear/aerogear-android-sdk
+* iOS (Native) - https://github.com/aerogear/aerogear-ios-sdk
+* Cordova (JS) - https://github.com/aerogear/aerogear-js-sdk
```
Comment (josemigallas) [inline sdks/sdk_features_matrix.md:8]: What about React Native and Xamarin? Are they not still in?

Comment (david-martin) [reply sdks/sdk_features_matrix.md:8]: As per Wei's comment https://github.com/aerogear/proposals/pull/33#discussion_r176776523, I'll probably move this from proposals repo.
This is more of a 'state of the world' doc that what's commited to or on the roadmap.
Xamarin would likely be added and filled out in time as it's done

**Comment by danielpassos** [comment]
@david-martin the file is `.md` but you are using asciidoc format?

**Comment by danielpassos** [comment]
@david-martin Is this still valid?

**Pull request closed without merging**